easybuild/easyconfigs/b/Blender/Blender-3.3.1.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PackedBinary'
+
+name = 'Blender'
+version = '3.3.1'
+versionsuffix = ''
+
+homepage = 'https://www.blender.org/'
+description = """Blender is the free and open source 3D creation suite. It supports
+ the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering,
+ compositing and motion tracking, even video editing and game creation."""
+
+toolchain = {'name': 'system', 'version': ''}
+
+sources = ['blender-%(version)s-linux-x64.tar.xz']
+source_urls = ['https://ftp.nluug.nl/pub/graphics/blender/release/Blender%(version_major_minor)s/']
+patches = []
+checksums = [
+    '3089a485dd621785d7a702089aba72d07b8f733a362e901ec1449b9a379546f2'
+]
+
+builddependencies = []
+
+dependencies = [
+    ('CUDA', '11.7.0')
+]
+
+sanity_check_paths = {
+    'files': ['blender', 'blender-softwaregl'],
+    'dirs': ['3.3']
+}
```

Comment:
```suggestion
sanity_check_paths = {
    'files': ['%(namelower)s'],
    'dirs': ['%(version_major_minor)s'],
}

modaliases = {
    'blender': 'blender -- --cycles-device CUDA',
}
```

Reply:
>  'blender': 'blender -- --cycles-device CUDA'

This is a really weird thing to do, as it forces CUDA to be used for rendering, while OPTIX can usually by a lot faster. So having this alias might force a lower performance. Also, I don't see why you would override the setting that the user has chosen in the Blender preferences?

Reply:
@paulmelis Agreed. I have removed that suggestion.